Messenger/AppDelegate.mm

```diff
@@ -118,6 +118,7 @@ - (void)applicationDidFinishLaunching:(NSNotification*)notification {
   webView.preferences = wp;
   auto req = [[NSURLRequest alloc] initWithURL:[NSURL URLWithString:@"https://www.messenger.com/t/"]];
   [webView.mainFrame loadRequest:req];
+  [webView setContinuousSpellCheckingEnabled:YES];
```

Comment:
@rsms I've never done objective-c before, so I'm not entirely sure what you mean? If you mean like `webView.setContinuousSpellCheckingEnabled = YES;` I get an error because it's not a property.